Rmarkdown/Multiple_Ascending_Dose_PD_continuous.Rmd

```diff
@@ -1,314 +1,311 @@
+gg <- gg + xgx_stat_ci(conf_level = .95)
+
+gg <- gg + xgx_scale_x_time_units(units_dataset = time_units_dataset, 
+                                  units_plot    = time_units_plot)
+gg <- gg + guides(color=guide_legend(""),fill=guide_legend(""))
+gg <- gg + scale_x_continuous(breaks=seq(-1,9,1))
+gg <- gg + scale_y_continuous()
+gg <- gg + labs(y=pd_label)
+gg <- gg + xgx_annotate_status(status)
+
+#if saving copy of figure, replace xgx_annotate lines with xgx_save() shown below:
+#xgx_save(width, height, dirs, "filename_main", status)
+print(gg)
+
+```
+
+### PD marker over time, faceted by Dose, mean (95% CI) percentiles by nominal time
+
+```{r, cache = TRUE, fig.width = 10, fig.height = 3, warning = FALSE, message= FALSE}
+ 
+gg <- ggplot(data = pd_data %>% subset(DOSE>0), 
+               aes(x=NOMTIME/24,y=LIDV)) 
+gg <- gg + xgx_stat_ci(conf_level = .95)
+gg <- gg + xgx_annotate_status(status)
+gg <- gg + xgx_scale_x_time_units(units_dataset = time_units_dataset, 
+                                  units_plot    = time_units_plot)
+gg <- gg + facet_grid(~TRTACT_low2high)
+gg <- gg + guides(color=guide_legend(""),fill=guide_legend(""))
+gg <- gg + scale_x_continuous(breaks=seq(-1,9,1))
+gg <- gg + scale_y_continuous()
+gg <- gg + labs(y=pd_label)
+
+#if saving copy of figure, replace xgx_annotate lines with xgx_save() shown below:
+#xgx_save(width, height, dirs, "filename_main", status)
+print(gg)
+
+```
+
+## Explore variability
+
+Use spaghetti plots to visualize the extent of variability between individuals. The wider the spread of the profiles, the higher the between subject variability. Distinguish different doses by color, or separate into different panels. If coloring by dose, do the individuals in the different dose groups overlap across doses? Dose there seem to be more variability at higher or lower concentrations?
+
+
+### PD marker over time, colored by Dose, dots & lines grouped by individuals
+
+```{r, cache = TRUE, fig.width = 8, fig.height = 4, warning = FALSE, message= FALSE}
+ 
+ gg <- ggplot(data = pd_data %>% subset(DOSE>0), 
+               aes(x=TIME/24, y=LIDV,group=ID, color =factor(TRTACT_high2low))) 
+ gg <- gg + xgx_annotate_status(status)
+ gg <- gg + geom_line()
+ gg <- gg + guides(color=guide_legend(""),fill=guide_legend(""))
+ gg <- gg + xgx_scale_x_time_units(units_dataset = time_units_dataset, 
+                                   units_plot    = time_units_plot)    
+ gg <- gg + scale_x_continuous(breaks=seq(-1,9,1))
+ gg <- gg + scale_y_continuous()
+ gg <- gg + labs(y=pd_label)
+ gg <- gg + xgx_annotate_status(status)
+
+#if saving copy of figure, replace xgx_annotate lines with xgx_save() shown below:
+#xgx_save(width, height, dirs, "filename_main", status)
+ print(gg)
+
+```
+
+### PD marker over time, faceted by Dose, dots & lines grouped by individuals
+
+```{r, cache = TRUE, fig.width = 10, fig.height = 3, warning = FALSE, message= FALSE}
+ 
+gg <- ggplot(data = pd_data %>% subset(DOSE>0), 

```

Comment:
don't need to filter out placebo, don't need to scale time

Reply:
ok all changed now